### ISSUE TITLE
Fails some pads

### ISSUE THREAD

**Issue by choll2000** [wrl/maschine.rs#5]
Fails pads 7, 8, 11 and 16

**Comment by wrl** [comment]
Fails how?

**Comment by choll2000** [comment]
I have Maschine Mikro mk2. 
It is difficult to explain,  it behaves randomly. Pad 8 and 7 works after pressing several times. The rest of the pads 11 and 16 fails when I have previously pressed 7 or 8. Red color of pads 7 and 8 and sometimes 11 lights stronger as if it was pressed. 
Thank you. Sorry my bad english.

**Comment by wrl** [comment]
It's quite alright. The pad handling in Maschine.rs isn't great right now and it's on my list to refactor/fix it.

Does the Maschine Mikro work correctly when using the official NI software on Mac or Windows?

**Comment by choll2000** [comment]
Yes, in windows works perfectly. I update my comment, Red color of pads 7 and 8 and sometimes 11 lights stronger as if it was pressed.

**Comment by wrl** [comment]
Alright, I'll take a look into it. Thanks for letting me know.

**Comment by choll2000** [comment]
Thank you, I appreciate it. I work on linux and I dont want to change.

**Comment by choll2000** [comment]
I have found out that if I put maschine vertically it works perfectly, the weight of the pads causes a certain pulsation. The problem seems to be in calculating offset of each pad and threshold.
As I indicated before, maschine works perfectly in windows.
Thank you.
